### ISSUE TITLE
Issue: Float labels in 2 item selection

### ISSUE THREAD

**Issue by gitnob** [zynthian/zynthian-issue-tracking#1067]
# Describe the issue
"Select" knob labels for 2 selection choices in for example "Add Special Chain" shows float numbers.
## Steps to reproduce the behavior:
There is a fix with in [oram (199670b)](https://github.com/zynthian/zynthian-ui/commit/199670bf888184f37f3ba0ed04b6d1fe5fa80f13) , but it's kind of a work around.

The fix was still there in https://github.com/zynthian/zynthian-ui/commit/ec8de5495bf01d7d6f9e2100160cf21480269b7c,
but was reverted until https://github.com/zynthian/zynthian-ui/commit/b87aad90672a8dbbd97cc146cec2597d7316733f .
 ## Expected behaviour
Integer choices for 2 selection objects.

# Additional context

# Configuration
## Hardware
```
Raspberry Pi 4 Model B Rev 1.2
Audio: HifiBerry DAC+ ADC PRO
Display: ZynScreen 3.5 (v1)
Wiring: MCP23017_ZynScreen
I2C: MCP23017@0x20
Profile: v4_stage
```
## System
```
Debian GNU/Linux 12 (bookworm)
Build Date: 2024-02-01
Memory: 12% (471M/3792M)
SD Card: 24% (13G/58G)
Temperature: 56.0ºC
Overclock: None
```
## MIDI & UI
```
Tuning: 440 Hz
Master Channel: Off
Preload Presets: On
ZS3 (SubSnapShots): On
Power Save: 60 minutes
Audio Levels on Snapshots: On
```
## Software
```
zyncoder: oram (a7976d8) 
zynthian-ui: oram (b87aad9) - this is after the update, unfortunately.
zynthian-sys: oram (d542728)
zynthian-data: oram (db0355a)
zynthian-webconf: oram (c27fea2)
```

**Comment by riban-bw** [comment]
The issue report is confusing. The first commit reference is unrelated. (It points to the removal of vcgencmd.) The next two refrences imply that a fix was applied then removed but in fact the fix was applied in [d7316733f](https://github.com/zynthian/zynthian-ui/commit/b87aad90672a8dbbd97cc146cec2597d7316733f).

I agree there is more work to do here to restore the control of small float parameters but I believe this issue is now resolved and that the detail in the report is flawed / confusing.

**Comment by gitnob** [comment]
oh, sorry, you fixed it too fast :-) also I didn't know what to write after your fix together with a hint to jofemodo. Didn't want to be confusing.
But it's closed now - fine :-)
